app/models/webhooks/slack/motion_closed.rb

```diff
@@ -15,4 +15,8 @@ def attachment_text
   def attachment_fields
     [view_motion_on_loomio]
   end
+
+  def attachment_color
```

Comment:
This looks great guys.

My last nitpick: moving this empty attachment_color up to the `base.rb` class, so we can only define / overwrite it when we need to.